crosschain-resolver/contracts/IMetadataResolver.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+interface IMetadataResolver {
+    /**
+     * @notice Get metadata about the CCIP Resolver
+     * @dev This function provides metadata about the CCIP Resolver, including its name, coin type, GraphQL URL, storage type, and encoded information.
+     * @param name The domain name in format (dnsEncoded)
+     * @return name The name of the resolver ("CCIP RESOLVER")
+     * @return coinType Resolvers coin type (60 for Ethereum)
```

Comment:
What does it mean for a resolver to have a coin type?

Reply:
The cointype of the chain the target contract (aka the l2 resolver) locates such as OP, Base, Arb, etc